### PR TITLE
adding postsubmit bazel job

### PR DESCRIPTION
**Pull request by RagniGarg97**
post-submit job to build bazel image.
https://github.com/ppc64le-cloud/test-infra/blob/master/images/bazel/Dockerfile

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/132#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@RagniGarg97: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | 33aaddf63c94ec4f9c98f4e8f63f1b7fc81745cf | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/132/pull-test-infra-yamllint/1359496602250645504) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=132). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:RagniGarg97).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by ltccci** [comment]
@RagniGarg97: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | 98bc5a8e0483ddb8d7330ce7c50eb22a3e932049 | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/132/pull-test-infra-yamllint/1359497186382974976) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=132). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:RagniGarg97).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (Rajalakshmi-Girish) [inline config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:12]: We might not need to run this job with `build-img` service account from IKS as we are not framing the KUBECONFIG in job.
@mkumatag please confirm.

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (mkumatag) [reply config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:12]: Make sense..

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:33]: if you look at here: https://github.com/ppc64le-cloud/test-infra/blob/master/images/bazel/Makefile#L10, we are basically building 3 different version of it, lets build all those combinations instead of `:latest` tag

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (Rajalakshmi-Girish) [inline config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:23]: https://github.com/ppc64le-cloud/test-infra/blob/master/images/all-in-one/Dockerfile#L113
has the `$GOPATH/bin` to the `PATH` variable. Please check if this is needed.
Also, the all-in-one image seems to have `GO111MODULE=on`.

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:34]: bazel version is missing here, you are supposed to pass the bazel version to the docker build arg, refer https://github.com/ppc64le-cloud/test-infra/blob/master/images/bazel/Makefile#L11 for more information.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml:34]: True!

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/132#issuecomment-780313606" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/132#" title="Author self-approved">RagniGarg97</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@RagniGarg97: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `bazel-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/132):

>post-submit job to build bazel image.
>https://github.com/ppc64le-cloud/test-infra/blob/master/images/bazel/Dockerfile


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
